src/routes/api/comment/comments.route.js

```diff
@@ -6,8 +6,11 @@ import confirmEmailAuth from '../../../middlewares/emailVarification.middleware'
 
 const router = express.Router();
 router.post('/:slug', [auth, CommentsValidation], Comments.createComment);
+router.get('/:articleSlug', Comments.getAllCommentsOfArticle);
 router.get('/', [auth], Comments.getComments);
 router.delete('/:id', [auth], Comments.deleteComment);
+router.delete('/particularArticle/:id', [auth], Comments.deleteCommentOfAnArticle);
```

Comment:
I don't think delete have a relationship with delete, so you might still use the current delete comment endpoint